frontend/src/components/MapView/Map/index.tsx

```diff
@@ -171,7 +171,6 @@ const MapComponent = memo(
       (map: Map) => {
         // Track with local state to minimize expensive dispatch call
         const layerIds = new Set<LayerKey>();
-        map.on('sourcedataloading', mapSourceListener(layerIds));
```

Comment:
Not usefull because we already have sourcedata listener.
And make a lot of rerenders...
![image](https://github.com/WFP-VAM/prism-app/assets/100352902/4827b26a-13af-44ed-bf2d-26489c08ddab)